plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -114,6 +114,15 @@ public void testCreateTableWithColumnComment()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Override
+    public void testCreateTableWithColumnCommentSpecialCharacter(String comment)
+    {
+        // TODO: Enable this test after fixing the issue
+        assertThatThrownBy(() -> super.testCreateTableWithColumnCommentSpecialCharacter(comment))
+                .hasMessageContaining("expected [%s] but found [Accumulo row ID]".formatted(comment));
+        throw new SkipException("Accumulo connector ignores column comments when creating a new table");
```

Comment:
Do we have a bug issue for this?

Reply:
https://github.com/trinodb/trino/issues/14095